pkg/backuphelpers/backupvars.go

```diff
@@ -0,0 +1,52 @@
+package backuphelpers
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	backupv1alpha1 "github.com/openshift/api/config/v1alpha1"
+)
+
+type BackupVar interface {
+	SetBackupSpec(spec backupv1alpha1.EtcdBackupSpec)
```

Comment:
another thing we can add here is the listener we also have in EnvVar:

https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/etcdenvvar/envvarcontroller.go#L36

so you can hook into the sync queue in the targetconfigcontroller:
https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L73

that way we can directly notify, instead of waiting for the 1m resync period. Not a game breaker for now, but maybe if you want to have a follow-up PR.

Reply:
ok i do it now

Reply:
fixed 👍🏽